src/models/community_business.ts

```diff
@@ -602,6 +603,27 @@ export const CommunityBusinesses: CommunityBusinessCollection & CustomMethods =
             } , {});
           return { age };
         }),
+      lastWeek: applyQueryModifiers(client('visit')
+        .select({ createdAt: 'visit.created_at' })
+        .whereRaw('visit.created_at >= CURRENT_DATE - INTERVAL \'7 day\'')
+        .innerJoin('visit_activity', 'visit_activity.visit_activity_id', 'visit.visit_activity_id')
```

Comment:
What are all these joins for? Doesn't look like they're used.

Reply:
These are used if there is a query passed in, see line L510-L519